src/main/java/de/funfried/netbeans/plugins/external/formatter/FormatterService.java

```diff
@@ -9,20 +9,17 @@
  */
 package de.funfried.netbeans.plugins.external.formatter;
 
+import de.funfried.netbeans.plugins.external.formatter.exceptions.FormattingFailedException;
```

Comment:
I can't see any real code changes in this file only reformating of the packages and the JavaDoc, so please revert that file as well

Reply:
Well, I basically run the pre-configured External Formatter Plugin over all the code.
Reverted these changes.

Reply:
You did not revert the changes 😄 
Otherwise the changes would have disappeared from the PR, but again, I'll handle it 👍